src/ServicePulse.Host/app/css/particular.css

```diff
@@ -1576,23 +1578,30 @@ span.metadata.danger, i.fa.fa-trash-o.danger, sp-moment.danger {
         float: none;
         margin: 15px 0;
     }
-    /* since 3.1.0 */
+    
     .navbar-collapse.collapse.in {
         display: block !important;
+        padding: 0 32px 16px 32px;
     }
 
     .collapsing {
         overflow: hidden !important;
     }
 }
 
-@media only screen and (min-width: 1200px) {
-    .container {
-        width: 92%;
-    }
+.table-head-row .col-xl-7, .endpoint-row .col-xl-7 {
+    width: 20% !important;
```

Comment:
Which style are these competing with that they need `!important`?

Reply:
Must have been needed before. No longer needed.